src/main/java/com/minecolonies/entity/EntityFishHook.java

```diff
@@ -735,7 +699,7 @@ private void spawnLootAndExp(@NotNull final EntityCitizen citizen)
      * @param citizen the fisherman getting the loot
      * @return an ItemStack randomly from the loot table
      */
-    private ItemStack getFishingLoot(@NotNull final EntityCitizen citizen)
+    private ItemStack getFishingLoot(final EntityCitizen citizen)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getFishingLoot" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)